draft-ietf-rats-corim.md

```diff
@@ -490,6 +492,27 @@ The role of "the CoRIM signer" MUST be specified in both cases, with the same co
 
 It is out of scope of this document to specify a method of delegating the signer role in the case that an unsigned CoRIM is conveyed through multiple secured links with different notions of authenticity without end-to-end integrity protection.
 
+### CoRIM bundles
+
+A method of signing a bundle of CoRIMs together is through a signed RATS Conceptual Message Wrapper (CMW) {{-cmw}}.
+Let `tag:{{&SELF}}:bundle` name a collection type that when signed MUST include the `corim-meta` protected header.
+The `corim-meta` header ensures that each CoRIM in the bundle has an identified signer.
+
+The CMW MAY use any label for its CoRIMs.
```

Comment:
The following section seems too terse to follow. Possibly, more of the CDDL context is needed?

Reply:
Added.